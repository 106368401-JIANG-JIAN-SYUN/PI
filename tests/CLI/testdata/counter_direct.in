table_add ExactOne 0 => actionA 0
counter_reset ExactOne_counter 9
counter_read ExactOne_counter 9
counter_write ExactOne_counter 9 packets=7 bytes=9993
counter_read ExactOne_counter 9
counter_write ExactOne_counter 9 packets=12
counter_read ExactOne_counter 9
counter_reset ExactOne_counter 9
counter_read ExactOne_counter 9
