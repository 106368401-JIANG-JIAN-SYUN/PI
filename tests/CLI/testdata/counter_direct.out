????
Device assigned successfully.
Selecting device.
????
Entry was successfully added with handle 0.
????
????
Dumping counter entry:
	Packets: 0
	Bytes: 0
????
Dumping counter entry:
	Packets: 7
	Bytes: 9993
????
Dumping counter entry:
	Packets: 7
	Bytes: 9993
????
Dumping counter entry:
	Packets: 12
????
Dumping counter entry:
	Packets: 12
	Bytes: 9993
????
????
Dumping counter entry:
	Packets: 0
	Bytes: 0
????
